test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/BodyModelBinderProviderTest.cs

```diff
@@ -70,11 +71,28 @@ public void GetBinder_WhenBindingSourceIsFromBody_ReturnsBinder()
             Assert.IsType<BodyModelBinder>(result);
         }
 
+        [Fact]
+        public void ConstructorWithoutLoggerFactory_DoesNotThrowNullReferenceException()
```

Comment:
nit: Actually testing that `GetBinder()` doesn't `throw`. Suggest renaming to reflect that.